static/app/views/profiling/utils.tsx

```diff
@@ -35,3 +35,15 @@ export function getColorEncodingFromLocation(location: Location): ColorEncoding
 
   return 'interaction_name';
 }
+
+export function generateFlamegraphRoute({
```

Comment:
I would just put this into the renderTableBodyCell. We are not reusing it anywhere and idk if utils is the right place for routing related code. Alternatively, it would probably make more sense to have something like views/profiling/routes.tsx where we define all the routes for profiling and we can have code like this live there.